### PR TITLE
Fixing iMX Build

### LINKED ISSUE #27400: [Platform] iMX builds are failing very often
### Reproduction steps

See this log: https://github.com/project-chip/connectedhomeip/actions/runs/5338205022?pr=27392

iMX docker images seem to be larger than GitHub runners can handle (17 GB)

Likely going to disable this in the interim, as it's failing lots of PRs/builds.	

### Platform

IMX8

### Platform Version(s)

latest

### Type

Manually tested with SDK

### (Optional) If manually tested please explain why this is only manually tested

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by woody-apple**
Trying to reduce the size of iMX

Fixes https://github.com/project-chip/connectedhomeip/issues/27400

### CODE REVIEW

File: .github/workflows/docker_img.yaml [hunk 1]
```diff
@@ -42,11 +42,6 @@ jobs:
                     - "-esp32"
                     - "-esp32-qemu"
                     - "-infineon"
-                    # NOTE: imx image requires too much space for GitHub-hosted runners. It fails with:
```
Comment (bzbarsky-apple) [inline .github/workflows/docker_img.yaml:45]: Why remove this comment?

File: integrations/docker/images/chip-build/version [hunk 2]
```diff
@@ -1 +1 @@
-0.7.18 Version bump reason: vscode ZEPHYR SDK env path update
+0.7.19.1 Version bump reason: Reducing size of imx
```
Comment (bzbarsky-apple) [inline integrations/docker/images/chip-build/version:1]: Why not just 0.7.19 or 0.7.20?

Comment (woody-apple) [reply integrations/docker/images/chip-build/version:1]: To not conflict with another going on from Andrei

File: .github/workflows/docker_img.yaml (hunk 1 above)
Comment (woody-apple) [reply .github/workflows/docker_img.yaml:45]: Because I had it working, then it broke again, and didn’t want to re-spin CI… I’ll fix and force in.

Comment (woody-apple) [reply .github/workflows/docker_img.yaml:45]: Fixed

File: integrations/docker/images/chip-build/version (hunk 2 above)
Comment (andy31415) [inline integrations/docker/images/chip-build/version:1]: Why not 0.7.20?

Comment (andy31415) [reply integrations/docker/images/chip-build/version:1]: For the future I would vote for 3 digit only, to make our regex history easier. Conflicts are ok and intended.

**Pull request merged**